### ISSUE TITLE
How does the indent rule work? I always get errors no matter what formatting.

### ISSUE THREAD

**Issue by trusktr** [BenoitZugmeyer/eslint-plugin-html#105]
I'm trying to learn how the indent rule work. I have some code that I think is formatted well, but I get an error. This my config:

```js
	settings: {
		'html/html-extensions': ['.html'],
		'html/indent': 'tab',
		'html/report-bad-indent': 'error',
	},
```

And I get this error:

<img width="454" alt="screen shot 2018-10-27 at 5 42 48 pm" src="https://user-images.githubusercontent.com/297678/47610658-c5a3a180-da0f-11e8-8aa8-17927920ef42.png">

<img width="422" alt="screen shot 2018-10-27 at 5 42 56 pm" src="https://user-images.githubusercontent.com/297678/47610659-c805fb80-da0f-11e8-9188-bfddf742d5a7.png">

<img width="403" alt="screen shot 2018-10-27 at 5 43 50 pm" src="https://user-images.githubusercontent.com/297678/47610662-e23fd980-da0f-11e8-855f-4427d3eddf4c.png">

(same error on that last one).

If I change the config to `'html/indent': '+2',`, then I get the same error on all lines:

<img width="425" alt="screen shot 2018-10-27 at 5 45 50 pm" src="https://user-images.githubusercontent.com/297678/47610673-31860a00-da10-11e8-9c31-a289266d17fd.png">

What does this option do exactly?

**Comment by BenoitZugmeyer** [comment]
Sorry for the late answer. Is the [documentation](https://github.com/BenoitZugmeyer/eslint-plugin-html#htmlindent) not clear enough?
* a number designates a number of spaces
* `tab` designates a single tab as indentation
* Starting with `+` indicates the indentation to be relative to the indentation of the opening `<script>` tag

So: in your first configuration, you are using `"tab"`. The first JS line should be indented with a single tab, no space. Ex:
```html
»	»	<script>
»	const $ = require('jquery')
»	...
```

Using `"+tab"`, you'll need to indent your code relatively to the `<script>` tag, ex:
```html
»	»	<script>
»	»	»	const $ = require('jquery')
»	»	»	...
```

By using `"+2"`, you'll need to indent your code with 2 spaces relatively to the `<script>` tag, ex:
```html
········<script>
··········const $ = require('jquery')
··········...
```

Let me know if it is not clear.

**Comment by trusktr** [comment]
@BenoitZugmeyer Hi, thanks for the reply. I understand what you are saying, but it isn't doing that. It is not complaining or giving me any error when the indentation is not as specified. So when I have bad indentation, I don't see any red error marks in my Atom editor. I do see red marks for other problems, but not these ones.

Am I thinking about these wrong? Do these show lint errors?
